portfolio/src/main/webapp/script.js

```diff
@@ -150,3 +151,15 @@ function toggleImgs(element, containerName) {
     $(element).addClass("fa-sort-up").removeClass("fa-sort-down") : 
     $(element).addClass("fa-sort-down").removeClass("fa-sort-up");
 }
+
+/**
+ * Checks the login status of the user. If the user isn't logged in, hides the comment form
+ */
+async function checkLogin() {
+  const resp = await fetch('/login');
+  const json = await resp.json();
+  console.log(json);
+  if (!json) {
+    $("#form-container").addClass("invisible");
```

Comment:
Instead of "hide the comments", it might be better to "show the comments only if the users is logged in".

If for some reasons, /login has a high latency, users will see the comments show up first and disappear later. That is typically a janky experience.

If you "show the comments async", it is better because it is basically "lazy load comments, which is not needed for the initial page".